galaxy_ng/app/api/ui/viewsets/namespace.py

```diff
@@ -44,57 +39,18 @@ def keywords_filter(self, queryset, name, value):
 class NamespaceViewSet(
     mixins.ListModelMixin,
     mixins.RetrieveModelMixin,
+    mixins.CreateModelMixin,
     mixins.UpdateModelMixin,
-    api_base.GenericViewSet,
+    viewsets.GenericViewSet,
```

Comment:
Using base classes from `galaxy_ng.app.api` is necessary, because they implement custom Authentication, Pagination and Error Handling.